extensions/azurecore/src/azureResource/providers/database/databaseService.ts

```diff
@@ -19,18 +20,27 @@ export class AzureResourceDatabaseService implements IAzureResourceService<azure
 		const resourceClient = new ResourceGraphClient(credential, { baseUri: account.properties.providerSettings.settings.armResource.endpoint });
 
 		// Query servers and databases in parallel (start both promises before waiting on the 1st)
-		let serverQueryPromise = queryGraphResources<GraphData>(resourceClient, subscriptions, serversQuery);
+		let synapseQueryPromise = queryGraphResources<GraphData>(resourceClient, subscriptions, synapseWorkspacesQuery);
+		let serverQueryPromise = queryGraphResources<GraphData>(resourceClient, subscriptions, sqlServersQuery);
 		let dbQueryPromise = queryGraphResources<GraphData>(resourceClient, subscriptions, `where type == "${azureResource.AzureResourceType.sqlDatabase}"`);
-		let servers: DbServerGraphData[] = await serverQueryPromise as DbServerGraphData[];
+		let server1: (DbServerGraphData | DbSynapseGraphData)[] = await serverQueryPromise as DbServerGraphData[];
+		let server2: (DbServerGraphData | DbSynapseGraphData)[] = await synapseQueryPromise as DbSynapseGraphData[];
```

Comment:
Same as above, except this should be `synapseWorkspaces` and `DbSynapseGraphData`